library/globals.inc.php

```diff
@@ -1372,6 +1372,12 @@
       '',                               // default
       xl('Email address, if any, to receive emergency login user activation messages.')
     ),
+     'new_validate' => array(
```

Comment:
Others have not been doing this, but I prefer adding "gbl_" in front of these names to avoid name collisions .  So 'gbl_new_validate'.